src/irmin-pack/unix/gc.mli

```diff
@@ -17,7 +17,8 @@
 open! Import
 
 (** [Make] returns a module that can manage GC processes. *)
-module Make (Args : Gc_args.S) : sig
+module Make
+    (Args : Gc_args.S) : sig
```

Comment:
This is worse :P (but much less important than the corrected functor argument layout)

Reply:
Noted.

Reply:
Skipping a version is fine. Though, that's bigger diffs for next time.